src/game/server/gamecontext.cpp

```diff
@@ -4526,4 +4526,4 @@ void CGameContext::Converse(int ClientID, const char* pStr)
 		Msg.m_pMessage = FinalMessage.buffer();
 		Server()->SendPackMsg(&Msg, MSGFLAG_VITAL, pPlayer->m_LastWhisperTo);
 	}
-}
```

Comment:
i don't understand what was fixed here :-) it seems that the compilation error that i posted was not fixed at all

Reply:
ok, the error is a little different
```
[52/77] #1 c++ src/game/server/gamecontext.cpp
src/game/server/gamecontext.cpp: In member function ‘virtual void CGameContext::OnMessage(int, CUnpacker*, int)’:
src/game/server/gamecontext.cpp:1791:41: error: invalid conversion from ‘const char*’ to ‘char*’ [-fpermissive]
     Converse(ClientID, pMsg->m_pMessage + 10);
                        ~~~~~~~~~~~~~~~~~^~~~
In file included from src/game/server/gamecontext.cpp:11:0:
src/game/server/gamecontext.h:267:7: note:   initializing argument 2 of ‘void CGameContext::Converse(int, char*)’
  void Converse(int ClientID, char *pStr);
       ^~~~~~~~
src/game/server/gamecontext.cpp:1795:41: error: invalid conversion from ‘const char*’ to ‘char*’ [-fpermissive]
     Converse(ClientID, pMsg->m_pMessage + 3);
                        ~~~~~~~~~~~~~~~~~^~~
In file included from src/game/server/gamecontext.cpp:11:0:
src/game/server/gamecontext.h:267:7: note:   initializing argument 2 of ‘void CGameContext::Converse(int, char*)’
  void Converse(int ClientID, char *pStr);
       ^~~~~~~~
src/game/server/gamecontext.cpp: In member function ‘void CGameContext::Converse(int, char*)’:
src/game/server/gamecontext.cpp:4533:10: error: return-statement with a value, in function returning 'void' [-fpermissive]
   return false;
          ^~~~~
src/game/server/gamecontext.cpp:4545:47: error: ‘Buffer’ was not declared in this scope
   TextIter = FinalMessage.append_at(TextIter, Buffer.buffer());
                                               ^~~~~~
bam: 'objs/game/server/gamecontext_sql.o' error 256
```

Reply:
look at the commit "Fix: fix bugs"